docs/reference/search/search.asciidoc

```diff
@@ -552,8 +553,8 @@ Period of time used to extend the life of the PIT.
 
 [[request-body-rank]]
 `rank`::
-(Optional, object) Defines a method for combining and ranking `1` standard query
-with `1..n` knn searches or no standard query with `2..n` knn searches.
+(Optional, object) Defines a method for combining and ranking result sets from
+a query and/or multiple knn searches.
```

Comment:
I think this might be misleading. To me it can say "a query OR multiple kNN searches", implying you can run it on a single query without a kNN search. Maybe it would be better to say "... ranking result sets from multiple queries, such as a query and kNN search or multiple kNN searches" ?

Reply:
I changed this completely. Hopefully, it's clear what the options are now.